src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -0,0 +1,308 @@
+      return (
+        <div>
+          {openModalButton}
+          <Modal
+            id={"story-book-modal"}
+            title={"Modal title"}
+            triggerElement={ref.current}
+            show={show}
+            onClose={closeModal}
+            isAlertDialog
+            closeButtonAriaLabel={"close"}
+          >
+            <ModalContent>
+              <form>
+                <TextField title="Your Name" autoFocus size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+                <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+                <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+                <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+                <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              </form>
+            </ModalContent>
+            <ModalFooter>{modalButtons}</ModalFooter>
+          </Modal>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Size variants
+
+<Canvas>
+  <Story name="Size variants - Normal">
+    {() => {
+      const { openModalButton, modalButtons, show, closeModal, ref } = useModalHelper({
+        openButtonTitle: "Default"
+      });
+      return (
+        <div>
+          {openModalButton}
+          <Modal
+            id={"story-book-modal"}
+            title={"Modal title"}
+            triggerElement={ref.current}
+            show={show}
+            onClose={closeModal}
+            closeButtonAriaLabel={"close"}
+            size={Modal.Size.DEFAULT}
+          >
+            <p>Normal modal</p>
+            <ModalFooter>{modalButtons}</ModalFooter>
+          </Modal>
+        </div>
+      );
+    }}
+  </Story>
+  <Story name="Size variants - Full">
+    {() => {
+      const { openModalButton, modalButtons, show, closeModal, ref } = useModalHelper({
+        openButtonTitle: "Full width"
+      });
+      return (
+        <div>
+          {openModalButton}
+          <Modal
+            id={"story-book-modal"}
+            title={"Modal title"}
+            triggerElement={ref.current}
+            show={show}
+            onClose={closeModal}
+            size={Modal.Size.FULL_WIDTH}
+            closeButtonAriaLabel={"close"}
+          >
+            <p>Full width modal</p>
+            <ModalFooter>{modalButtons}</ModalFooter>
+          </Modal>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Use cases and examples
+
+### Modal with vertical scroll
+
+The height of Modal with long content will be limited to 90% of screen height, and a scroll bar will be added
+
+<Canvas>
+  <Story name="Modal with vertical scroll">
+    {() => {
+      const { openModalButton, modalButtons, show, closeModal, ref } = useModalHelper({
+        openButtonTitle: "Default"
```

Comment:
It's better to change name of button "Default" - to something more related to story: "Vertical scroll"